src/main/resources/library/attribute-definitions.json

```diff
@@ -9,7 +9,7 @@
     "library:numSubjects", "library:projectName", "library:dataCategory", "library:datatype", "library:dataUseRestriction",
     "library:studyDesign", "library:cellType", "library:requiresExternalApproval"],
   "oneOf": [{
-    "required": ["library:GRU", "library:HMB", "library:NCU", "library:NPU", "library:NDMS", "library:NAGR",
+    "required": ["library:GRU", "library:HMB", "library:NCU", "library:NPU", "library:NDMS",
```

Comment:
where did you get that NAGR is not required? It shows as required on the schema spreadsheet.

Moran expressed that there *is* a difference between the user explicitly choosing "Unspecified" and the user not answering the question. So maybe this one can't be a boolean.